frontend/www/js/omegaup/arena/navigation.ts

```diff
@@ -37,7 +38,7 @@ export type NavigationRequest =
   | NavigationForContest
   | NavigationForSingleProblemOrCourse;
 
-export function navigateToProblem(request: NavigationRequest): void {
+export function navigateToProblem(request: NavigationRequest): Promise<void> {
```

Comment:
ya arreglé todas las pruebas para referencia futura. lo único que faltaría es:

aquí esto se puede declarar como

```suggestion
export async function navigateToProblem(request: NavigationRequest): Promise<void> {
```

y con ese cambio L56 necesitaría cambiarse a `return;` y L87-L89 se debería eliminar. eso funciona?